src/tabs/status.rs

```diff
@@ -0,0 +1,29 @@
+use super::prelude::*;
```

Comment:
If we're refactoring, how about adding spacing between import groups like `cargo fmt` would do by default?

Reply:
This is a personal preference. I always use `cargo fmt` but without a special config and no spaces were added.
So maybe `cargo fmt` on a nightly channel behaves different?